docs/topics/collections-overview.md

```diff
@@ -8,7 +8,7 @@ collections, you can skip this introduction and proceed to the detailed sections
 
 A collection usually contains a number of objects (this number may also be zero) of the same type. Objects in a collection
```

Comment:
Again, if we want to be accurate, change this to state "of the same type (and its subtypes)".